src/templates/security-scheme-template.js

```diff
@@ -401,7 +401,7 @@ export default function securitySchemeTemplate() {
           <div class="blue-text"> ${providedApiKeys.length} API key applied </div>
           <div style="flex:1"></div>
           <button class="m-btn thin-border" part="btn btn-outline" @click=${() => { onClearAllApiKeys.call(this); }}>CLEAR ALL API KEYS</button>`
-        : html`<div class="red-text">No API key applied</div>`
+        : html`<div class="red-text">No API key applied - <a href="#auth">Apply one here</a></div>`
```

Comment:
`#auth` wont be available always.it depends on which section the user is showing and also special care must be taken for focused view